src/test/datascience/notebook/ipywidget.vscode.test.ts

```diff
@@ -4,97 +4,79 @@
 'use strict';
 
 /* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
-// import * as path from 'path';
-// import * as sinon from 'sinon';
+import * as path from 'path';
```

Comment:
Code change was easy, but the only place that webviews property that I moved from NotebookProvider to KernelProvider was being used was here in these widget tests. And these widget tests were actually being skipped. So I change them to use the new code, and re-enabled them, and fixed up a few issues with them.

Were running on my local machine with this change, will check in CI on the PR.